src/transformers/file_utils.py

```diff
@@ -94,7 +94,7 @@
 DUMMY_MASK = [[1, 1, 1, 1, 1], [1, 1, 1, 0, 0], [0, 0, 0, 1, 1]]
 
 S3_BUCKET_PREFIX = "https://s3.amazonaws.com/models.huggingface.co/bert"
-CLOUDFRONT_DISTRIB_PREFIX = "https://d2ws9o8vfrpkyk.cloudfront.net"
+CLOUDFRONT_DISTRIB_PREFIX = "https://cdn.huggingface.co"
```

Comment:
Do we want to have a `models` in the CDN URL?

Reply:
I'd say not necessarily, but I'm open to changing it

Reply:
Mostly that I can picture in the future some models having the same names as datasets or other types of data. But there are other solutions as well I guess.

Reply:
Well, one cloudfront distribution (i.e. domain) will only work for one S3 bucket anyways, so I was thinking datasets would be `cdn-datasets.huggingface.co` or something like that

Reply:
I guess this one could be `cdn-models` but i think we're fine with that one

Reply:
👍